tasks/update-infra-deployments/update-infra-deployments.yaml

```diff
@@ -3,24 +3,23 @@ apiVersion: tekton.dev/v1
 kind: Task
 metadata:
   labels:
-    app.kubernetes.io/version: "0.5.0"
+    app.kubernetes.io/version: "0.5.1"
```

Comment:
I think we should go to 1.0.0 here - removing a parameter is a breaking change

Reply:
ACK, thanks!